app/main.py

```diff
@@ -2,0 +1,35 @@
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "mv":
+        raise ValueError("Bad command!")
+    source = parts[1]
+    destination = parts[2]
+
+    if not os.path.isfile(source):
+        raise FileNotFoundError(f"{source} not found.")
+
+    if destination.endswith("/"):
+        destination = os.path.join(destination, os.path.basename(source))
+    destination_directory = os.path.dirname(destination)
+
+    if not os.path.exists(destination_directory):
+        os.makedirs(destination_directory, exist_ok=True)
+
+    elif (os.path.exists(destination_directory)
+          and not os.path.isdir(destination_directory)):
+        raise FileExistsError(f"{destination_directory}"
+                              f" exist, but is not directory.")
```

Comment:
The error message here has a grammatical issue. It should be 'exists' instead of 'exist'. Consider changing the message to: `f"{destination_directory} exists, but is not a directory."`.